src/scripts.js

```diff
@@ -15,12 +15,16 @@ const recipeSection = document.getElementById('allRecipes');
 const modalSection = document.getElementById('recipeModalBackground');
 const filterDropdown = document.getElementById('filterDropdown');
 const searchBar = document.getElementById('searchBar')
+let modalRecipe 
+// const saveButton = document.getElementById('saveBtn')
```

Comment:
Love the global modalRecipe. Let's remember to delete this unused line of code below it too!